source/client/output_formatter_impl.cc

```diff
@@ -292,7 +312,12 @@ std::string FortioOutputFormatterImpl::formatProto(const nighthawk::client::Outp
   fortio_output.set_numthreads(number_of_connections);
 
   // Get the result that represents all workers (global)
-  const auto& nh_global_result = getGlobalResult(output);
+  auto nh_global_result_status = getGlobalResult(output);
+  if (!nh_global_result_status.ok()) {
+    return absl::Status(nh_global_result_status.status().code(),
```

Comment:
This could be covered in a test as well, I think

Reply:
Even if we create a test with input that doesn't contain a global result, it gets handled at https://github.com/envoyproxy/nighthawk/blob/a8d5fdef25d6039a39adfd9944e1bf294a53b667/source/client/output_formatter_impl.cc#L291

therefore, the lines @oschaaf mentioned remain uncovered.

Reply:
That makes me wonder if it's even possible to hit this code.. if that's not the case it means this is dead code, and we ought to remove it. IMHO that would make sense and is related to why I figured that throwing an exception in this case didn't make sense. It looks like the Fortio UI would indicate an error on the output we'd produce, which is sufficient:
https://github.com/fortio/fortio/blob/ee098c572115f4149a67d2a812cb3d1ca79e30e1/ui/static/js/fortio_chart.js#L105

We should probably make sure that this really works though.